locations/spiders/kfc_my.py

```diff
@@ -0,0 +1,29 @@
+import scrapy
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+
+class KFCMYSpider(scrapy.Spider):
+    name = "kfc_my"
+    item_attributes = {"brand": "KFC", "brand_wikidata": "Q524757"}
+    allowed_domains = ["kfc.com.my"]
+    start_urls = [
+        "https://kfc.com.my/graphql?query=query+allLocation%7BallLocation%7Blocations%7Baddress+city+code+coleslaw+country+created+curbside+delivery_close+delivery_open+delivery_tier+dinein+drivethru+gesStoreId+is_breakfast+is_delivery+is_selfcollect+lat+launch_date+legacy_store_id+locationId+long+name+phone+riderType+selfcollect_close+selfcollect_open+selfcollect_tier+polygon+smartbox+state+storeCmgId+storeName+updated+zip+disabled_skus+is_breakfast+coleslaw+drivethru+smartbox+dinein+__typename%7D__typename%7D%7D&operationName=allLocation&variables=%7B%7D"
+    ]
```

Comment:
So I've been playing with graphql today. This request is actually:

```graphql
query allLocation {
    allLocation {
        locations {
            address
            city
            code
            coleslaw
            country
            created
            curbside
            delivery_close
            delivery_open
            delivery_tier
            dinein
            drivethru
            gesStoreId
            is_breakfast
            is_delivery
            is_selfcollect
            lat
            launch_date
            legacy_store_id
            locationId
            long
            name
            phone
            riderType
            selfcollect_close
            selfcollect_open
            selfcollect_tier
            polygon
            smartbox
            state
            storeCmgId
            storeName
            updated
            zip
            disabled_skus
            is_breakfast
            coleslaw
            drivethru
            smartbox
            dinein
            __typename
        }
    __typename
    }
}
```

We can remove any fields from that which we don't need. Removing `disabled_skus` halves the response size. `coleslaw` is listed twice :p

 We can also rename fields here, instead of cleaning up after dictparser we could:
```patch
-            locationId
+            ref: locationId
```

Reply:
Fixed :)